eta-company-calendar-backend/src/main/java/hu/flowacademy/companycalendar/service/MeetingService.java

```diff
@@ -0,0 +1,68 @@
+package hu.flowacademy.companycalendar.service;
+
+import hu.flowacademy.companycalendar.model.Meeting;
+import hu.flowacademy.companycalendar.model.dto.MeetingDTO;
+import hu.flowacademy.companycalendar.repository.MeetingRepository;
+import hu.flowacademy.companycalendar.repository.UserRepository;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+@Builder
+@Transactional
+@AllArgsConstructor
+public class MeetingService {
+
+    private final MeetingRepository meetingRepository;
+    private final UserRepository userRepository; // TODO - UserService is not ready yet
+
+    public List<MeetingDTO> findAll() {
+       return meetingRepository
+               .findAll()
+               .stream()
+               .map(MeetingDTO::new)
+               .collect(Collectors.toList());
+    }
+
+    public MeetingDTO findOne(Long id) {
+        return new MeetingDTO(meetingRepository.findById(id)
+                .orElseThrow(() -> new ResponseStatusException(HttpStatus.NOT_FOUND, "Meeting not found")));
```

Comment:
[major] Ne adjunk vissza a service rétegben controllerhez kapcsolódó objectet. Ezt inkább kezeljük a controllerben.

Reply:
Robival abban maradt a csapat, hogy egységesen a service-k DTO-t fogadnak és adnak vissza, máskülönben az átalakításhoz a controllerbe be kellene húzni a UserRepository-t.
Vagy nem erre gondolsz?

Reply:
Javítva!